src/components/alert/Alert.stories.js

```diff
@@ -0,0 +1,104 @@
+import UidsAlert from './Alert.vue';
+
+// More on default export: https://storybook.js.org/docs/vue/writing-stories/introduction#default-export
+export default {
+  title: 'Components/Alert',
+  component: UidsAlert,
+  // More on argTypes: https://storybook.js.org/docs/vue/api/argtypes
+  argTypes: {
+    type: {
+      name: 'Type',
+      options: ['info', 'success', 'warning', 'danger'],
+      control: {
+        type: 'select',
+        labels: {
+          'info': 'Info',
+          'success': 'Success',
+          'warning': 'Warning',
+          'danger': 'Danger',
+        },
+      },
+      table: {
+        category: 'Display options',
+      },
+    },
+    centered: {
+      name: 'Centered',
+      table: {
+        category: 'Display options',
+      },
+    },
+    dismissible: {
+      name: 'Dismissible',
+      table: {
+        category: 'Properties',
+      },
+    },
+    verticallyCentered: {
+      name: 'Center Icon Vertically',
+      table: {
+        category: 'Display options',
+      },
+    },
+    default: {
+      name: 'Content',
+      control: { type: 'text' },
+      table: {
+        category: 'Content',
+      },
+    },
+  },
+};
+
+// More on component templates: https://storybook.js.org/docs/vue/writing-stories/introduction#using-args
```

Comment:
```suggestion
// More on component templates: https://storybook.js.org/docs/vue/writing-stories/introduction#using-args
```
Let's remove this for actual components.